catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/CswSource.java

```diff
@@ -384,7 +384,7 @@ public void refresh(Map<String, Object> configuration) throws SecurityServiceExc
 
         String idProp = (String) configuration.get(ID_PROPERTY);
         if (StringUtils.isNotBlank(idProp)) {
-            cswSourceConfiguration.setId(idProp);
+            setId(idProp);
```

Comment:
So why did this one line change, but all the rest of the properties are still getting set in the cswSourceConfiguration object? It also looks like cswSourceConfiguration.getId() is called later on in this method, which would now presumably return the incorrect value. Something doesn't seem right about this change.